src/typings/worker.ts

```diff
@@ -136,7 +136,7 @@ type WorkerRespMap = {
         numPages: number,
         pageNum: number,
     },
-    [WORKER_RESP_CODE.QUERY_RESULT]: { results: QueryResults },
+    [WORKER_RESP_CODE.QUERY_RESULT]: { progress: number, results: QueryResults },
```

Comment:
This should be automatically fixed by linters once we enable TS specific syntax checks in `eslint-config-yscope`.
```suggestion
    [WORKER_RESP_CODE.QUERY_RESULT]: {
        progress: number,
        results: QueryResults
    },
```